helpers/DeployHelper.ts

```diff
@@ -297,6 +295,31 @@ export class DeployHelper {
       state,
     };
   }
+
+  async upgradeValidator(
+    validatorAddress: string,
+    validatorContractName: string,
+  ): Promise<{
+    validator: Contract;
+  }> {
+    this.log("======== Validator: upgrade started ========");
+
+    const owner = this.signers[0];
+
+    this.log("upgrading validator...");
+    const ValidatorFactory = await ethers.getContractFactory(validatorContractName);
+    const validator = await upgrades.upgradeProxy(validatorAddress, ValidatorFactory, {
+      unsafeSkipStorageCheck: true
```

Comment:
Why we skip storage checks and don't even make it optional?

Reply:
it is our script, what is the sense to make it optional on this stage?